tools/tools.mk

```diff
@@ -54,5 +54,11 @@ else
 	curl -LsS https://github.com/gohugoio/hugo/releases/download/v$(HUGO_VERSION)/hugo_extended_$(HUGO_VERSION)_macOS-64bit.tar.gz | tar zxv -C $(TOOLS_DIR)/hugo
 endif
 
+# fio
+fio:
+	sudo apt-get -qq update
```

Comment:
This assumes a Debian-based environment.
It seems that the installation of all the other tools work on any Linux or macOS environment.

Reply:
OK, we'll see if there's any pushback upstream.

Reply:
I've added a Linux condition around the install, so at least it won't fail the tools install segment for macOS